src/main/java/ru/yandex/practicum/filmorate/storage/FilmStorage.java

```diff
@@ -0,0 +1,17 @@
+package ru.yandex.practicum.filmorate.storage;
+
+import org.springframework.web.bind.annotation.RequestBody;
+import ru.yandex.practicum.filmorate.model.Film;
+
+import javax.validation.Valid;
+import java.util.Collection;
+
+public interface FilmStorage {
+    public Collection<Film> findAll();
```

Comment:
Все методы интерфейсов по умолчанию имеют модификатор доступа `public`, поэтому повторно определять его не нужно. Удали, пожалуйста, `public` во всех методах интерфейсов, где он присутствует

```suggestion
    List<Film> findAll();
```